lang/en/docs/software-directory/machine-learning/python-ml/components.md

```diff
@@ -16,21 +16,216 @@ available executables and flavors under the
 PythonML is based on the `python` [executable](../../../software/components.md#executables), and through this executable
 the implemented ML calculations can be performed.
 
+## Workflow Structure
```

Comment:
(1) Maybe let's create a subfolder in `images/` for `python-ml` and keep the images there?
(2) the naming of the files is inconsistent - some are CamelCase, some snake_case, some with dashes - is there a convention? If not, let's establish it and explain in the README.md at the top level

Reply:
Moved the tutorial images to their own folder, and expanded the image section in the documentation. It now discusses the casing for the files, and how to name the file, its title, and its alt text.